TombEngine/Game/Lara/lara_struct.h

```diff
@@ -12,13 +12,13 @@ struct ItemInfo;
 using namespace TEN::Entities::Player;
 
 // Inventory object constants
-constexpr int NUM_PUZZLES		  = ID_PUZZLE_ITEM16 - ID_PUZZLE_ITEM1 + 1;
```

Comment:
This formatting was on purpose, please don't remove it.

Reply:
Not sure why VS removed it. Have added it back.